### PR TITLE
970: Updated copyright year

### PR DESCRIPTION
**Pull request by oruam85**
Updated copyright year

### CODE REVIEW

**Comment by dmarkov** [comment]
Let me find a reviewer for this pull request, thanks for submitting it

**Comment by dmarkov** [comment]
@krzyk it's yours, please review

File: src/site/apt/respect-limits.apt.vm [hunk 1]
```diff
@@ -7,7 +7,7 @@
  ------
 
 ~~
-~~ Copyright (c) 2014, jcabi.com
+~~ Copyright (c) 2015, jcabi.com
```
Comment (krzyk) [inline src/site/apt/respect-limits.apt.vm]: should be 2014-2015

**Comment by krzyk** [comment]
@oruam85 one comment above

**Comment by oruam85** [comment]
@krzyk Please check again.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor merge this

**Comment by dmarkov** [comment]
@krzyk Much obliged! I've added **17 mins** to your account in payment "50663600"

+17 to your rating, your total score is [+2733](http://www.netbout.com/b/35035?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**